nav2_bt_navigator/src/navigators/navigate_to_pose.cpp

```diff
@@ -82,14 +82,6 @@ NavigateToPoseNavigator::cleanup()
 bool
 NavigateToPoseNavigator::goalReceived(ActionT::Goal::ConstSharedPtr goal)
 {
-  auto bt_xml_filename = goal->behavior_tree;
-
-  if (!bt_action_server_->loadBehaviorTree(bt_xml_filename)) {
```

Comment:
Readd, I don't know why this was removed. that removes support for changing BTs dynamically on request.